index.js

```diff
@@ -485,8 +489,11 @@ function getMetadata (opts) {
     system: {
       hostname: opts.hostname,
       architecture: process.arch,
-      platform: process.platform
-    }
+      platform: process.platform,
+      container: undefined,
+      kubernetes: undefined
+    },
+    labels: opts.globalLabels
```

Comment:
I couldn't find any documentation for this, so just double check that the root of the payload is the right place to add the new `labels` property 😅

Reply:
I checked the apm-server PR and the spec files. This is the right place for it. :)